frontend/simple/views/components/GroupsMinIncome.vue

```diff
@@ -1,24 +1,37 @@
 <template>
-  <section class="has-text-right">
-    <p class="min-income-label has-text-grey">Min Income</p>
-    <p class="min-income title is-1 is-marginless"
-      data-test="minIncome">
-      {{ currency }}{{ group.incomeProvided }}
-    </p>
-    <a href="#">Propose change</a>
-  </section>
+  <div>
+    <div class="has-text-right">
+      <p class="min-income-label has-text-grey">Min Income</p>
+      <p class="min-income title is-1 is-marginless"
+        data-test="minIncome">
+        {{ currency }}{{ group.incomeProvided }}
+      </p>
+      <a href="#" @click="openProposal">
+        <i18n>Propose change</i18n>
+      </a>
+    </div>
+  </div>
```

Comment:
It seems like there's an unnecessary extra `<div>` on the outside of the `<div>`?